easybuild/easyconfigs/s/SciPy-bundle/SciPy-bundle-2020.02-foss-2020a-rc3-Python-3.8.2.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'PythonBundle'
+
+name = 'SciPy-bundle'
+version = '2020.02'
```

Comment:
Should we call this 2020a too?

Reply:
We base this on the release date of the most recent extension (which we also do for other bundles).

It gives some freedom in case someone needs a more recent bundle for whatever reason (which we wouldn't use as a dep in the `2020a` generation, but it could still be useful).

That doesn't work if we use `2020a` as a version...

Reply:
Ok, just checking.